include/fixed_containers/optional_reference.hpp

```diff
@@ -43,6 +43,15 @@ class OptionalReference
     {
     }
 
+    constexpr OptionalReference(pointer ptr)
```

Comment:
We've chosen to mark the ctors as explicit. See the comment below. New ctors should keep this pattern.

Reply:
Done.